scripts/init.sql

```diff
@@ -0,0 +1,30 @@
+create table gift_certificate
+(
+    id bigint auto_increment,
+    name varchar(50),
+    description varchar(50),
+    price decimal(10, 2) default 0 null,
+    duration int,
+    create_date varchar(50),
+    last_update_date varchar(50),
```

Comment:
Your DB should have types for representing date, time, datetime data. Please use suitable type, but no varchar